.github/ISSUE_TEMPLATE/other.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Other
+about: Anything that doesn't neatly fit into the "bug" or "feature request" templates. DO NOT post support requests here.
```

Comment:
At first glance, it is read as `Anything that doesn't neatly fit into the "bug" or "feature request"`
```suggestion
about: Anything that doesn't neatly fit into the listed templates. DO NOT post support requests here.
```

Reply:
I'm not completely convinced this is clearer so have left it for now